Sources/MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -4,65 +4,85 @@ import MapboxCoreNavigation
-}
-
-public extension InstructionsBannerViewDelegate {
-    /**
-     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
-     */
-    func didTapInstructionsBanner(_ sender: BaseInstructionsBannerView) {
-        logUnimplemented(protocolType: InstructionsBannerViewDelegate.self, level: .debug)
-    }
+ A banner view that contains the current step instruction and responds to tap and swipe gestures.
+ 
+ This class responds and gets updated as the user progresses along a route according to the `NavigationComponent` and `BaseInstructionsBannerView` protocol.
+*/
+@IBDesignable
+open class InstructionsBannerView: BaseInstructionsBannerView, NavigationComponent {
     
     /**
-     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
+     Updates the instructions banner info as the user progresses along a route.
+     
+     - parameter service: The `NavigationService` instance that passes the instruction.
+     - parameter instruction: The `VisualInstructionBanner` instance to be presented.
+     - parameter routeProgress: The`RouteProgress` instance that the instruction banner view is updating.
      */
-    func didSwipeInstructionsBanner(_ sender: BaseInstructionsBannerView, swipeDirection direction: UISwipeGestureRecognizer.Direction) {
-        logUnimplemented(protocolType: InstructionsBannerViewDelegate.self, level: .debug)
-    }
-}
-
-/// :nodoc:
-@IBDesignable
-open class InstructionsBannerView: BaseInstructionsBannerView, NavigationComponent {
     public func navigationService(_ service: NavigationService, didPassVisualInstructionPoint instruction: VisualInstructionBanner, routeProgress: RouteProgress) {
         update(for: instruction)
     }
 }
 
-/// :nodoc:
+/**
+ A banner view that contains the current step instruction along a route and responds to tap and swipe gestures, as the base of `InstructionsCardView` and `InstructionsBannerView`.
+*/
 open class BaseInstructionsBannerView: UIControl {
+    
+    /**
+     A view that contains a simple image indicating a type of maneuver.
+     */
     public weak var maneuverView: ManeuverView!
+    
+    /**
+     A primary instruction label indicates the current step.
+     */
     public weak var primaryLabel: PrimaryLabel!
+    
+    /**
+     A secondary instruction label below the `primaryLabel` to provide detail information of current step.
+     */
     public weak var secondaryLabel: SecondaryLabel!
+    
+    /**
+     A styled lable indicates the remaining distance along the current step.
+     */
     public weak var distanceLabel: DistanceLabel!
+    
+    /**
+     A vertical view divides between `maneuverView` and `distanceLabel` to `primaryLabel` and `secondaryLabel`.
+     */
     public weak var dividerView: UIView!
     weak var _separatorView: UIView!
+    
+    /**
+     A invisible helper view for visualizing the result of the constraints.
```

Comment:
```suggestion
     An invisible helper view for visualizing the result of the constraints.
```

Not sure why we ended up making it public. As it doesn't seem to be really useful for the end user.

Reply:
Thanks for the suggestion, the spelling error fix and improvements are made.